releasecalendar/data.go

```diff
@@ -0,0 +1,49 @@
+package releasecalendar
+
+// Release represents a release
+type Release struct {
```

Comment:
Should you include metadata fields like:

```go
LastUpdated       time.Time                         `json:"last_updated,omitempty"`
```

`omitempty` if the field doesn't exist, but once releases are ported over into document db, I would expect this field to be set. Maybe this is future work

Reply:
Another field would be ETag - https://github.com/ONSdigital/dp-dataset-api/blob/develop/models/instance.go#L25